docs/root/version_history/current.rst

```diff
@@ -136,4 +136,4 @@ Deprecated
   :ref:`contains_match <envoy_v3_api_field_config.route.v3.HeaderMatcher.contains_match>` are deprecated by :ref:`string_match <envoy_v3_api_field_config.route.v3.HeaderMatcher.string_match>`.
 * listener: :ref:`reuse_port <envoy_v3_api_field_config.listener.v3.Listener.reuse_port>` has been
   deprecated in favor of :ref:`enable_reuse_port <envoy_v3_api_field_config.listener.v3.Listener.enable_reuse_port>`.
-  At the same time, the default has been changed from false to true. See above for more information.
+  At the same time, the default is getting changed from false to true. See above for more information.
```

Comment:
i think the original is probably more correct here

Reply:
Reverted.